domain-models-runtime/src/test/java/org/folio/rest/persist/PgUtilIT.java

```diff
@@ -402,6 +402,109 @@ public void deleteByIdResponseWithout204(TestContext testContext) {
         asyncAssertSuccess(testContext, 500, "respond204"));
   }
 
+  @Test
+  public void deleteByCQLwithNo500(TestContext testContext) {
+    PostgresClient pg = PostgresClient.getInstance(vertx, "testtenant");
+    //  insert(testContext, pg, "delete_test",  1);
+    PgUtil.delete("users",  "username=delete_test",
+        okapiHeaders, vertx.getOrCreateContext(), ResponseWithout500.class,
+        asyncAssertFail(testContext, "respond500"));
+  }
+
+  @Test
+  public void deleteByCQLwithNo400(TestContext testContext) {
+    PgUtil.delete("users", "username=delete_test",
+        okapiHeaders, vertx.getOrCreateContext(), ResponseWithout400.class,
+        asyncAssertSuccess(testContext, 500, "respond400"));
+  }
+
+  @Test
+  public void deleteByCQLwithNo204(TestContext testContext) {
+    PgUtil.delete("users", "username=delete_test",
+        okapiHeaders, vertx.getOrCreateContext(), ResponseWithout204.class,
+        asyncAssertSuccess(testContext, 500, "respond204"));
+  }
+
+  @Test
+  public void deleteByCQLNullHeaders(TestContext testContext) {
+    PostgresClient pg = PostgresClient.getInstance(vertx, "testtenant");
+    insert(testContext, pg, "delete_test",  1);
+    PgUtil.delete("users",  "username==delete_test",
+        null, vertx.getOrCreateContext(), Users.DeleteUsersByUserIdResponse.class,
+        asyncAssertSuccess(testContext, 400, "null"));
+  }
+
+  @Test
+  public void deleteByCQLOK(TestContext testContext) {
+    PostgresClient pg = PostgresClient.getInstance(vertx, "testtenant");
+    insert(testContext, pg, "delete_test",  1);
```

Comment:
This test reuses the name "delete_test". This makes this test fail if the test deleteByCQLNullHeaders runs before this test:
```
[ERROR] deleteByCQLOK  Time elapsed: 0.009 s  <<< ERROR!
io.vertx.pgclient.PgException: { "message": "duplicate key value violates unique constraint \"users_pkey\"", "severity": "ERROR", "code": "23505", "detail": "Key (id)=(4d318819-8675-07ab-d95b-e877c5789410) already exists.", "file": "nbtinsert.c", "line": "427", "routine": "_bt_check_unique", "schema": "testtenant_raml_module_builder", "table": "users", "constraint": "users_pkey" }
```

A different name should be used.

Reply:
Deleting that name in use for NullHeaders